lib/public/js/main.js

```diff
@@ -61,5 +66,27 @@ const newsletterSubmit = () => {
     };
 };
 
+const scrollToElement = (selector) => {
```

Comment:
This is a pretty sweet little solution for now, but would love to remove it once we figure-out whatsup!